doc/development_guide/contributor_guide/contribute.rst

```diff
@@ -27,12 +27,9 @@ of Pylint as it gives you access to the latest ``ast`` parser.
   For more information on how to use our test suite and write new tests see :ref:`testing`.
 
 - ``pylint`` uses black_ and isort_ among other Python auto-formatters.
-  We have a pre-commit hook which should take care of the autoformatting for
-  you. To enable it, do the following:
 
-    * install ``pre-commit`` using ``pip install pre-commit``
-
-    * then run ``pre-commit install`` in the ``pylint`` root directory to enable the git hooks.
+  We have pre-commit hooks which should take care of the autoformatting for
+  you. To enable it, run ``pre-commit install --install-hooks -t pre-push -t pre-commit`` in the ``pylint`` root directory.
```

Comment:
If this is opt-in, then I guess it's alright. Daniel and I can opt-out. :D

Reply:
It's launched just before pushing not for every commit. I could also limit what trigger the check.